library/src/main/kotlin/me/thanel/swipeactionview/SwipeActionView.kt

```diff
@@ -471,16 +477,20 @@ class SwipeActionView : FrameLayout {
         animateToOriginalPosition(startDelay)
     }
 
+
     /**
      * Animate the view to its original position.
      *
      * @param startDelay The amount of delay, in milliseconds, to wait before starting the
      * movement animation. (Defaults to 0)
+     *
+     * @param completeCallback Will be run at the end of the animation. (Defaults to null)
      */
     @JvmOverloads
-    fun animateToOriginalPosition(startDelay: Long = 0) {
+    fun animateToOriginalPosition(startDelay: Long = 0, completeCallback: Runnable? = null) {
```

Comment:
I like this approach with completion but could you use kotlin's function type instead of Runnable?

```suggestion
    fun animateToOriginalPosition(startDelay: Long = 0, completeCallback: () -> Unit = {}) {
```

With documentation change that it by default does nothing instead of being null.